host/4/publish.yml

```diff
@@ -80,8 +80,13 @@ steps:
   - bash: |
       set -e
       docker pull $SOURCE_REGISTRY/dotnet:$(PrivateVersion)-dotnet8
+      docker pull $SOURCE_REGISTRY/dotnet:$(PrivateVersion)-dotnet8-appservice
```

Comment:
@Cooperlink should this be 8.0?

Reply:
I do not know it depends on what the platform is pulling for this stack. That is a good callout. I can check the code/ look for logs/ find an old linux dedicated stamp that is kicking around maybe

Reply:
This should be 8.0 to match the expected tag in Antares. Thank you Pragna

Reply:
Per discussion on Teams, it seems there would not be a difference in behavior from that, and other work will be needed to account for whatever form the tag takes. Therefore, the current form here should be equally fine.